apiserver/facades/agent/uniter/storage_test.go

```diff
@@ -123,8 +126,15 @@ func (s *storageSuite) TestWatchStorageAttachmentVolume(c *tc.C) {
 			return blockDevicesWatcher
 		},
 	}
+	applicationService := &mockApplicationService{
+		getUnitMachineName: func(ctx context.Context, unitName coreunit.Name) (coremachine.Name, error) {
+			calls = append(calls, "GetUnitMachineName")
+			c.Assert(unitName.String(), tc.DeepEquals, unitTag.Id())
+			return coremachine.Name(machineTag.Id()), nil
+		},
+	}
```

Comment:
Can you use a generated mock?

Reply:
I'd rather just skip the test here. Or make new tests. I'm not a fan of this method.